Examples/Tests/boundaries/analysis.py

```diff
@@ -101,9 +101,11 @@ def do_periodic(x):
 assert len(a_id) == 1, "Absorbing particles not absorbed"
 assert np.all(vx == -vx0), "Reflecting particle velocity not correct"
 assert np.all(vz == +vz0), "Periodic particle velocity not correct"
-assert np.all(np.abs((xx - xxa) / xx) < 1.0e-15), (
-    "Reflecting particle position not correct"
+rel_err_xx = np.abs((xx - xxa) / xx)
+assert np.all(rel_err_xx < 1e-9), (
+    f"Reflecting particle position along x not correct. Maximum relative error: {np.max(rel_err_xx)}"
 )
-assert np.all(np.abs((zz - zza) / zz) < 1.0e-15), (
-    "Periodic particle position not correct"
+rel_err_zz = np.abs((zz - zza) / zz)
+assert np.all(rel_err_zz < 1e-9), (
+    f"Periodic particle position along z not correct. Maximum relative error: {np.max(rel_err_zz)}"
```

Comment:
Increased the tolerance from `1e-15` to `1e-9` and added info to assert message for easier debugging in the future.

Reply:
Personally, I think that increasing all of the tolerances here is a not a good idea. I could agree that tolerances could be increased to ignore differences in round-off, but that's not the case here. We know why there are larger errors here, the inconsistency between the constants in WarpX and the values in scipy.constants. To do this correctly, the WarpX constants should be updated to be consistent, as in PR #5661. These two PR should be merged.

Reply:
Yes, I think that would work.